apps/platform-app/src/app/[locale]/(auth)/(components)/GithubLoginButton/GithubLoginButton.tsx

```diff
@@ -4,10 +4,12 @@ import { Button } from '@gdh/ui-system'
 import { GithubIcon } from '@gdh/ui-system/icons'
 import { signIn } from 'next-auth/react'
 import { useState } from 'react'
+import { type Roles } from '../../_models/User.model'
 import styles from './GithubLoginButton.module.scss'
 
```

Comment:
**praise**: Including role in the props enhances the component's flexibility and reusability.